### PR TITLE
TS-10 homepage template

### PR DESCRIPTION
**Pull request by john-ko**
added template.html as a guide

### CODE REVIEW

File: public_html/assets/css/style.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* HTML reset */
 html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+p, blockquote, pre,
```
Comment (john-ko) [inline public_html/assets/css/style.css:3]: removed to use header tags

File: public_html/assets/css/style.css [hunk 2]
```diff
@@ -41,13 +42,16 @@ time, mark, audio, video {
 	margin-right: 15px;
 	width: 305px;
 	float: left;
-	color: #992233;
 }
```
Comment (john-ko) [inline public_html/assets/css/style.css:45]: removed red color to thirds

File: public_html/assets/css/style.css [hunk 3]
```diff
@@ -80,21 +84,102 @@ img.splash {
...
+
+.header .header-logo-text{
+	border-top-style: solid;
+	border-bottom-style: solid;
+	background: #ffffff;
+	margin:auto;
+	text-align:center;
+	width:400px;
+	font-family: 'Playfair Display', serif;
+}
```
Comment (john-ko) [inline public_html/assets/css/style.css:190]: merged sam's css into style.css

File: public_html/templates/footer.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                        <div>
+                            <a href="https://www.instagram.com/kang_gary8888/">
+                                <img src="assets/images/instagram.png" width="20" height="auto"/>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </footer>
```
Comment (john-ko) [inline public_html/templates/footer.html]: merged kevin's footer

**Comment by sophiech** [comment]
looks good

**Pull request merged**